salt/etcd-proxy/etcd-proxy.conf.jinja

```diff
@@ -4,6 +4,19 @@
 ETCD_DATA_DIR="/var/lib/etcd/"
 ETCD_NAME="{{ grains['id'] }}"
 
+# Bug bsc#1043614, Cannot bootstrap cluster with the qcow2
+# Problem occured on virtualized environment with high network latency.
+# https://coreos.com/etcd/docs/latest/v2/tuning.html
+# The election timeout should be set based on the heartbeat 
+# interval and average round-trip time between members. 
+# Election timeouts must be at least 10 times the round-trip time 
+# so it can account for variance in the network.
+# The values are specified in milliseconds.
+# default ETCD_HEARTBEAT_INTERVAL=100
+ETCD_HEARTBEAT_INTERVAL=100
+# default ETCD_ELECTION_TIMEOUT=1000
+ETCD_ELECTION_TIMEOUT=2000
```

Comment:
This does not seem relevant to the bug in the commit.

From the logs in bsc#1043614 the error is 

```
curl -L -X PUT http://localhost:2379/v2/keys/_etcd/registry/k8s/_config/size -d value=2' failed with return code: 7
curl: (7) Failed to connect to localhost port 2379: Connection refused
``` 

when trying to interact with the discovery etcd, which is a single node cluster, and not managed by this config.

We only support a single region / DC install for CaaSP, so there should be no need to increase this (if the network latency is 200ms in the environment, there is a local environment issue)